src/scripting/vmthunks.cpp

```diff
@@ -3080,7 +3080,7 @@ static int GetRealTime()
 {
 	time_t now;
 	time(&now);
-	return now ? now : 0;
+	return now ? now : (time_t)(-1);
```

Comment:
In this form the comparison is useless. You need to return it as is (so -1 will designate an error), or to check `now` for `(time_t)(-1)` and return zero if it's the case (zero means an error). I would just do the first.